User/UserContext/schema.json

```diff
@@ -0,0 +1,69 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "id": "https://fiware.github.io/dataModels/User/UserContext/schema.json",
+  "title": "FIWARE - User Context schema",
+  "description": "Information on the context of an anonymized in a given point in time",
+  "type": "object",
+  "allOf": [
+    {
+      "$ref": "https://fiware.github.io/dataModels/common-schema.json#/definitions/GSMA-Commons"
+    },
+    {
+      "$ref": "https://fiware.github.io/dataModels/common-schema.json#/definitions/Location-Commons"
+    },
+    {
+      "properties": {
+        "refActivity": {
+          "$ref": "https://fiware.github.io/dataModels/common-schema.json#/definitions/EntityIdentifierType"
```

Comment:
for the sake of consistency refActivity could be a URI as well if we start using URIs for identifying different entities

Reply:
we should probably check also the existing datamodels (or re-define `EntityIdentifierType`)